src/converse-register.js

```diff
@@ -349,6 +371,9 @@
                         $form.append('<input type="button" class="submit" value="'+__('Return')+'"/>');
                         $form.find('input[type=button]').on('click', this.cancelRegistration.bind(this));
                     }
+                    if (_converse.registration_domain !== '') {
+                        $form.find('input[type=button]').hide();
+                    }
```

Comment:
Isn't this button now being omitted via the template? In which case this call is no longer necessary, right?